routing-updated/src/agent.js

```diff
@@ -2,15 +2,16 @@ const {
   ROUTER_ADDRESS,
   EVENT_SIGNATURE,
   createAlert,
-} = require('./agent.config');
+} = require("./agent.config");
 
 const handleTransaction = async (txEvent) => {
   const findings = [];
 
   const events = txEvent.filterLog(EVENT_SIGNATURE, ROUTER_ADDRESS);
-
+  const from = txEvent.from;
+  const fromToLowerCase = from.toLowerCase();
```

Comment:
We don't need to call toLowerCase because the txEvent.from has only lowercase letters